front/pages/api/w/[wId]/assistant/conversations/[cId]/events.ts

```diff
@@ -1,14 +1,28 @@
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import * as reporter from "io-ts-reporters";
 import { NextApiRequest, NextApiResponse } from "next";
 
 import { getConversation } from "@app/lib/api/assistant/conversation";
-import { getConversationEvents } from "@app/lib/api/assistant/pubsub";
+import {
+  cancelMessageGenerationEvent,
+  getConversationEvents,
+} from "@app/lib/api/assistant/pubsub";
 import { Authenticator, getSession } from "@app/lib/auth";
 import { ReturnedAPIErrorType } from "@app/lib/error";
 import { apiError, withLogging } from "@app/logger/withlogging";
 
+export type PostMessageEventResponseBody = {
+  success: true;
+};
+const PostMessageEventBodySchema = t.type({
```

Comment:
Let's make it another endpoint either `/action` or `/cancel` because it's weird to post to `/events` with a body that says `action: cancel` ?

Reply:
`/actions` I guess is more general and need to take an `s` :)

Reply:
Initially I thought it makes sense cause the goal of this endpoint is to generate a new cancellation event for this convo. I probably should have named it "type" instead of "action". 

I'll make the change anyway to post on /cancel!